widget/app.js

```diff
@@ -185,19 +186,15 @@
               var getCustomMarkerIcon = function (_imageUrl) {
                 return {
                   url: _imageUrl,
-                  // This marker is 20 pixels wide by 32 pixels high.
                   scaledSize: new google.maps.Size(20, 20),
-                  // The origin for this image is (0, 0).
-                  origin: new google.maps.Point(0, 0),
-                  // The anchor for this image is the base of the flagpole at (0, 32).
-                  anchor: new google.maps.Point(0, 32)
+                  anchor: new google.maps.Point(10, 10)
                 }
```

Comment:
this centers the pin on the location. pin is 20 X 20, centered on coordinates 10, 10